root/static/styles/tooltip.less

```diff
@@ -46,6 +46,7 @@
             background: @text-white;
             border: 1px solid @medium-border;
             box-shadow: 0 2px 5px @medium-border;
+            display: block;
```

Comment:
Can we use `inline-block` instead, or does that not display correctly?  Using display: block inside an inline element still seems odd even if it doesn't break anything.